docs/metrics-3.6.md

```diff
@@ -0,0 +1,81 @@
+# Metrics upgrade notes
+
+Metrics have changed between 3.5 and 3.6.
+
+You can now retrieve metrics using the OpenTelemetry Protocol using the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/), and this is the recommended mechanism.
+
+These notes explain the differences in using the prometheus `/metrics` endpoint to scrape metrics for a minimal effort upgrade. It is not recommended you follow this guide blindly, the new metrics have been introduced because they add value, and so they should be worth collecting and using.
+
+## TLS
+
+The prometheus `/metrics` endpoint defaults to TLS on.
+
+To disable this set `.metricsConfig.secure` to `false`.
+
+## New metrics
```

Comment:
hmm, this isn't a breaking change... we typically don't describe additions in the upgrading notes

Reply:
It is a breaking change for anyone who doesn't explicitly set `secure` in their configmap.

Reply:
I was commenting on the "New Metrics" section, not the TLS bit. It's also the longest section, so removing it would make the upgrade notes a lot shorter.

I think conventionally we just add new features to the existing docs (no specific upgrade note for them in the docs, though maybe in the release blog post) and add the "v3.6 and after" modifier where needed.

Reply:
New metrics will get consumed into your data collection tool by default, which may be costly. It feels important to warn people that they could be in for an increased bill by upgrading if they don't do something about it. I'd only expect people to read the upgrading guide.